CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.1.0 (July 13, 2020)
+
+### Internal Changes
+* Refactor use of `componentWillReceiveProps` to avoid React 16.9 deprecation warnings.
```

Comment:
I've since changed my thinking on changelogs and believe there's no need to add any internal refactoring lines to the changelog (e.g. devDependencies stuff), since it shouldn't change any behavior. Personally, I would call the removing of deprecation warnings a feature. What do you think?

Reply:
Also, I'm adding semantic-release to this package, which will automate the changelog, version bump and publishing for us. So, you actually don't need to add to the changelog here. Just format the title of the PR to match what type of release we want.

Reply:
>Personally, I would call the removing of deprecation warnings a feature.

Agree (I versioned to `2.1.0` to reflect that)

>Just format the title of the PR to match what type of release we want.

Updated title.